io.github.giantpinkrobots.bootqt.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.giantpinkrobots.bootqt
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-21.08
+
+command: bootqt.sh
+
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Flatpak
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+  
+modules:
+  - name: bootqt
+    buildsystem: simple
+    build-commands:
+      - install -D -m755 bootqt.sh /app/bin/bootqt.sh
+      - install -D -m755 bootqt.py -t /app/lib/bootqt
+      - install -D -m755 io.github.giantpinkrobots.bootqt.png -t /app/share/icons/hicolor/scalable/apps
+      - install -D -m755 io.github.giantpinkrobots.bootqt.desktop -t /app/share/applications
+    
+    sources:
+      - type: file
```

Comment:
why not checking out the whole repo?

Also you are missing an appstream file.

Reply:
I'm very sorry, I'm new to all this. The filesystem permission is because this is a bootable USB creator app. Do I put the appstream file right alongside the manifest file? I have it ready.

Reply:
You mean why not copy the whole repository? Because in the main repository I have separate Python scripts for standalone and flatpak versions. Thought copying the whole thing over would download a lot of unnecessary things.

Reply:
No. I mean a source type of `git`. Like virtually any other package getting source out of git.

Reply:
Understood. Now it is done. I also deleted the unnecessary things.